jwst/tests/base_classes.py

```diff
@@ -65,10 +65,15 @@ def get_data(self, *pathargs, docopy=True):
         `artifactory_helpers/get_bigdata()`.
         This will then return the full path to the local copy of the file.
         """
-        # If user has specified action for no_copy, apply it with
+        # If user has specified action for docopy use it, If none
         # default behavior being whatever was defined in the base class.
-        local_file = get_bigdata(*self.repo_path, *pathargs, docopy=self.docopy)
-
+#        if docopy is None: 
+#            local_file = get_bigdata(*self.repo_path, *pathargs, docopy=self.docopy)
+#        elif docopy:
+        local_file = get_bigdata(*self.repo_path, *pathargs, docopy=True)
```

Comment:
You want `docopy=docopy` here so it respects the API of `get_data()`.  Otherwise looks good.  Delete all the commented-out stuff.  👍